osquery/tables/specs/utility/file.table

```diff
@@ -20,6 +20,7 @@ schema([
     Column("is_link", INTEGER, "1 if a symlink else 0"),
     Column("is_char", INTEGER, "1 if a character special device else 0"),
     Column("is_block", INTEGER, "1 if a block special device else 0"),
+    Column("wildcard", TEXT, "A wildcard which can be used to match file patterns"),
```

Comment:
"wildcard" or "pattern"? I ask because wildcard doesn't sound right for some reason, it seems to singular.